packages/client/hmi-client/src/workflow/ops/simulate-ensemble-ciemss/tera-simulate-ensemble-node-ciemss.vue

```diff
@@ -78,6 +78,8 @@ const getStatus = async (simulationId: string) => {
 	emit('update-state', state);
 
 	if (pollerResults.state === PollerState.Cancelled) {
+		state.inProgressSimulationId = '';
+		emit('update-state', state);
 		return pollerResults;
 	}
```

Comment:
I think there is a better update pattern here, you are potentially calling `update-state` too much and that is generally not a a good thing.

Rather than

```
change_state_part_1
emit('update-state', ...)
if (blah) {
   change_state_part_2
   emit('update-state')
}
```

It would be better to have, if possible
```
change_state_part_1
if (blah) {
  change_state_part_2
}
emit('update-state')
```

Reply:
This is a good point. I will double check these files for this